### PR TITLE
Tribute home

### PR DESCRIPTION
**Pull request by mbonabucya**
Add Home page 
Add index.html
add main section 
add footer
Modify sytle.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                   
+                </li>
+
+            </ul>
+            <div id="achievement">
+               <p id="italicize"><q>Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress.</q></p>
+               <p>-- Indian Prime Minister Manmohan Singh</p>
+            </div>
+        </div>
+        <div id="tribute-link">
```
Comment (mbonabucya) [inline index.html:103]: Would you please consider to match the margin to the left with the design on free code camp

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
-# tribute
\ No newline at end of file
+# tribute
```
Comment (mbonabucya) [inline README.md]: Would please consider checking duplicated title in the README.md?
README.md should have one title

**Review by mbonabucya — COMMENTED** [review]
## STATUS: REQUEST CHANGES ♻️♻️

Hi @mbonabucya 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there!
143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91
## To highlight 🍾
Good commit history ✔️
Your design looks good 💯 ✔️
Good PR description ✔️
Good usage HTML sematics ✔️
 you have used well CSS selectors ✔️

## Required Changes ♻️♻️

- Check the comments under the review
- All linters should pass

## [ OPTIONAL ] 
For best practice, I suggest you using the section (header , body section and footer section)
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@mbonabucya ](https://github.com/mbonabucya) when you comment so I can receive the notification or use slack with the same name.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ]  [OPTIONAL] You've done a great job to address the issues a student needs to fix in a polite way and provided a reason it is important to fix them.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**